forge-gui/res/cardsfolder/upcoming/the_black_gate.txt

```diff
@@ -0,0 +1,10 @@
+Name:The Black Gate
+ManaCost:no cost
+Types:Legendary Land Gate
+K:ETBReplacement:Other:DBTap
+SVar:DBTap:DB$ Tap | ETB$ True | Defined$ Self | UnlessCost$ PayLife<3> | UnlessPayer$ You | StackDescription$ enters the battlefield tapped. | SpellDescription$ As CARDNAME enters the battlefield, you may pay 3 life. If you don't, it enters the battlefield tapped.
+A:AB$ Mana | Cost$ T | Produced$ B | SpellDescription$ Add {B}.
+A:AB$ ChoosePlayer | Cost$ 1 B T | Defined$ You | Choices$ Player.withMostLife | ChoiceTitle$ Choose a player with the most life or tied for most life | SubAbility$ DBEffect | SpellDescription$ Choose a player with the most life or tied for most life. Target creature can't be blocked by creatures that player controls this turn.
+SVar:DBEffect:DB$ Effect | ValidTgts$ Creature | RememberObjects$ ChosenPlayer,Targeted | StaticAbilities$ CantBlock | SubAbility$ DBCleanup
```

Comment:
needs cleanup

Reply:
wrong cleanup

Reply:
ClearChosenPlayer is not correct ?

Reply:
You mean some sort of hidden static trigger ?

Reply:
Don't do cleanup for something that's not needed